src/e2e/resources/simple-temple-expected/simple-temple-test-user/util/util.go

```diff
@@ -97,3 +97,13 @@ func ExtractAuthIDFromRequest(headers http.Header) (*Auth, error) {
 
 	return &Auth{uuid}, nil
 }
+
+// ExtractIDFromRequest extracts the parameter provided under parameter parent_id and converts it into a UUID
+func ExtractParentIDFromRequest(requestParams map[string]string) (uuid.UUID, error) {
+	id := requestParams["parent_id"]
+	if len(id) == 0 {
+		return uuid.Nil, errors.New("No Service ID provided")
```

Comment:
How come "No Service ID provided" as the error message?

Reply:
This is the ID of the parent service. What would you suggest @smithwjv?

Reply:
Ah I read too quick, my mistake

Reply:
Thought it was fetching the struct ID